src/io/tradetrust/bill-of-lading/1.0/BillOfLading.v3.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
I just realized the file name is not kebab cased. Please change this one

Did we do the same on the previous Mr? Let's change as well ( yeah i know the other are not kebab case but it's my fault and it's bad)

Reply:
done